test/java/seedu/addressbook/commands/FindCommandTest.java

```diff
@@ -0,0 +1,89 @@
+package seedu.addressbook.commands;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.util.TestUtil;
+import seedu.addressbook.util.TypicalPersons;
+
+/**
+ * Test class for the Find Command
+ */
+public class FindCommandTest {
+
+    private AddressBook addressBook;
+    
+    private final List<ReadOnlyPerson> EMPTY_LIST = Collections.emptyList();
+
+    @Before public void setUp() {
+        this.addressBook = new TypicalPersons().getTypicalAddressBook();
+    }
+
+    @Test
+    public void execute_sameWordSameCase_matched() throws IllegalValueException {
+        List<ReadOnlyPerson> expected = TestUtil.createList(new Person(TypicalPersons.amy));
+        
+        String[] keywordsWithMatchingCase = {"Amy"};
+        
+        FindCommand command = createFindCommand(keywordsWithMatchingCase);
+        assertCommandBehavior(command, expected, addressBook);
```

Comment:
Still too verbose. Out of this 4 lines, the only things that are specific to this test case is `Amy` and `amy`. Everything else is just boilerplate and can be pushed into the utility method.